### ISSUE TITLE
Train first MTL MLP model

### ISSUE THREAD

**Issue by scribbler00** [scribbler00/gefcom_scenario_forecast#3]
Train first MTL MLP model with 

- [x] aleatoric loss 
- [x] check if aleatoric loss function is novins
- [x] dropout rate of 0.3/ is it dropout rate or keep rate in keras?
- [x] weight decay of ??
